src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -209,6 +209,8 @@ public function load(array $configs, ContainerBuilder $container)
             if (!class_exists('Symfony\Component\Validator\Validation')) {
                 throw new LogicException('The Validator component is required to use the Form component.');
             }
+        } else {
+            $container->removeDefinition('Symfony\Component\Form\Command\DebugCommand');
```

Comment:
Thinking about future `DebugCommand` from other components ... mm I'm not sure if we want avoid collisions...